### PR TITLE
Rename Admin to System::Admin

### PR DESCRIPTION
**Pull request by allenwq**
Everything currently under `Admin::` has been moved to `System::Admin`

TODO:
Rename `SystemAnnouncement` model to `System::Announcement`
Rename `Admin::AnnouncementsController` to `System::Admin::InstanceAnnouncementsController`
Rename `Admin::SystemAnnouncementsController` to `System::Admin::AnnouncementsController`

### CODE REVIEW

**Comment by lowjoel** [comment]
The SystemAnnouncement model?

**Comment by allenwq** [comment]
Should `SystemAnnouncementController` be `System::AnnouncementController` ?  I'm not sure

**Comment by lowjoel** [comment]
Okay that would be a problem, it's fine then.

**Comment by lowjoel** [comment]
Or, maybe, you should call it System::Admin::AnnouncementsController and System::Admin::Instance::AnnouncementsController?

**Comment by lowjoel** [comment]
or InstanceAnnouncementsController?

**Comment by allenwq** [comment]
Ok, that would be a better alternation, I will use `InstanceAnnouncementsController` then.

**Comment by allenwq** [comment]
@lowjoel I've put the list of changes in the description

**Comment by allenwq** [comment]
\>< I think I got a bunch of conflicts to deal after the latest merge

**Comment by allenwq** [comment]
@lowjoel  I've done here

**Comment by allenwq** [comment]
@lowjoel Updated here, will do the announcements renamings in separate PRs.

File: app/views/announcements/index.html.slim [hunk 1]
```diff
@@ -1,5 +1,5 @@
 div.page-header
   h1
     = t('.header')
-= render partial: 'admin/announcements/announcement', collection: @announcements
+= render partial: 'system/admin/announcements/announcement', collection: @announcements
```
Comment (lowjoel) [inline app/views/announcements/index.html.slim]: The admin should use the global announcement partial, right?

Comment (allenwq) [inline app/views/announcements/index.html.slim]: Does global announcement have a partial ? I'm just updating the reference here.

Comment (lowjoel) [inline app/views/announcements/index.html.slim]: If so, it should appear in `/views/announcements/_announcement.html.slim` right?

Comment (allenwq) [inline app/views/announcements/index.html.slim]: I think currently global announcement is referencing the instance partial, you want the reverse way ?

**Comment by lowjoel** [comment]
@dariusf or @allenwq please move the partial to the right place.

**Pull request merged**